pages/client_page.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+    <div class="container">
+        <button>&lt;&ensp; Powrót</button>
+        <div class="client_profile">
+            <div>Profil użytkownika</div>
+            <img src="../assets/images/example_profile_picture.jpg" alt="Profile picture">
+        </div>
+        <div class="client_data">
+            <div class="data_header">Twoje dane</div>
+            <div class="username">janusz1968pl</div>
+            <div class="label_username">Nazwa użytkownika</div>
+            <div class="names">Janusz Nowacki</div>
+            <div class="label_names">Imię i nazwisko</div>
+            <div class="email">j.nowacki@p.cichowski.com</div>
+            <div class="label_email">E-mail</div>
+            <div class="subscriptions">
+                <div class="subscriptions_header">Subskrypcja i pakiety dodatkowe</div>
+                <div class="balance_state">
+                    Twój kolejny rachunek w dniu <strong>14.11.23</strong> wynosi <strong>30.56 zł</strong>.
+                </div>
+                <div class="bundles_label">Pakiety dodatkowe</div>
+                <div class="bundles">
+                    <div class="bundle">Pakiet 1</div>
+                    <div class="bundle">Pakiet 2</div>
+                </div>
+            </div>
+        </div>
+        <div class="buttons">
+            <div class="client_manage_button">
+                <div>Zarządzaj swoim panelem</div>
+            </div>
+            <div class="client_edit_button">
+                <div>Edytuj dane konta</div>
+            </div>
+        </div>
```

Comment:
przyciski powinny mieć pseudoklasy `:hover` i `:active` żeby wizualnie było widać że to przycisk (do stylu ale już zaznaczylem tu)